actioncable/test/connection/subscriptions_test.rb

```diff
@@ -66,6 +66,17 @@ def throw_exception(_data)
     end
   end
 
+  test "subscribe command with Base channel" do
```

Comment:
Not sure how valuable this test is. I can remove it if folks don't think it is useful.